src/engraving/rendering/dev/horizontalspacing.cpp

```diff
@@ -182,12 +182,14 @@ double HorizontalSpacing::minHorizontalDistance(const Segment* f, const Segment*
                 }
                 const EngravingItem* attachedLine = note->lineAttachPoints().front().line();
                 double minLength = 0.0;
-                if (attachedLine->isTie()) {
-                    minLength = f->style().styleMM(Sid::MinTieLength);
-                } else if (attachedLine->isGlissando()) {
-                    bool straight = toGlissando(attachedLine)->glissandoType() == GlissandoType::STRAIGHT;
-                    minLength = straight ? f->style().styleMM(Sid::MinStraightGlissandoLength)
-                                : f->style().styleMM(Sid::MinWigglyGlissandoLength);
+                if (attachedLine->visible()) {

```

Comment:
Here too, just put `if (!attachedLine->visible()) { continue; }`, because if it's not visible then the whole calculation is skippable